docs/all_development/common_tech/index.rst

```diff
@@ -0,0 +1,35 @@
+########################################
+Technology common to core and extensions
+########################################
+
+Information in the topics below is pertinent to both core and extension work.
+
+* `Analytics <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/analytics.html>`_
+* `Micro-frontends <https://edx.readthedocs.io/projects/edx-developer-docs/en/latest/developers_guide/micro_frontends_in_open_edx.html>`_
+* :doc:`Independently deployed applications (IDAs) <../../universal_stub>`
+* :doc:`Our data model <data_model>`
+* :doc:`Our messaging model <messaging_model>`
```

Comment:
@nedbat's comment moved from the closed PR https://github.com/edx/edx-developer-docs/pull/34/files#r564075611
> These two feel like they should be further down, perhaps in "Working with the core Open edX Platform". They are more detailed than the other links here, and aren't needed until you are going to write code in the platform.

Reply:
These topics have been moved to all_development/common_tech per the suggestion above.